.travis.yml

```diff
@@ -2,15 +2,16 @@ language: node_js
 node_js:
   - node
 cache: npm
-
 install:
   - npm ci
-
 script:
   - npm test
-
 deploy:
-  skip-cleanup: true
-  github-token: $GITHUB-TOKEN
```

Comment:
Why was the github token variable removed, which seems to be of utmost essence?

Reply:
It's not actually required for deployment from what I did earlier,
Actually, there's an automated Travis command to setup Heroku deployment,
```bash
$ travis setup heroku
```
It overrides the previous setting there.
![image](https://user-images.githubusercontent.com/36427779/80297049-a8bc6b80-8777-11ea-888f-82f77368b0ad.png)
Also from the image above, it shows that the key is unknown.

Reply:
The github token was setup as a Travis env. We will have to get a new one later, but for now, we will keep that on hold and continue with your implementation. After we have fully and successfully incorporated your implementation, we will then test the deployment with the new GitHub token. Well done @Holajuwon !

Reply:
The github token was setup as a Travis env. We will have to get a new one later, but for now, we will keep that on hold and continue with your implementation. After we have fully and successfully incorporated your implementation, we will then test the deployment with the new GitHub token. Well done @Holajuwon !